Symfony/CS/Tokenizer/Token.php

```diff
@@ -149,6 +145,25 @@ public function equalsAny(array $others, $caseSensitive = true)
     }
 
     /**
+     * A helper method used to find out whether or not a certain input token has to be case-sensitively matched.
+     *
+     * @param bool|bool[] $caseSensitive global case sensitiveness or an array of booleans, whose keys should match
+     *                                   the ones used in $others. If any is missing, the default case-sensitive
+     *                                   comparison is used.
+     * @param int|string  $key           the key of the token that has to be looked up
```

Comment:
It wont break anything: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.4/Symfony/CS/Tokenizer/Token.php#L117

Passing string as keys here are useless, don't suggest users to do so!